queries.sql

```diff
@@ -0,0 +1,19 @@
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Pikachu'; -----> this is a request
+SELECT * FROM animals; ---> to check whrther the updated item changed or not
+
+SELECT * FROM animals WHERE Name like '%mon';
+
+SELECT * FROM animals WHERE Date_of_birth >= '2016-01-01' AND Date_of_birth <= '2019-12-31';
```

Comment:
- [ ] Please make the change suggested by the previous reviewer about returning the names of animals. Please refer to the following query 👇 
```
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```